src/aurelia-view.js

```diff
@@ -1,14 +1,18 @@
 import {Config} from './config';
 import {ViewManager} from './view-manager';
+import {resolvedView} from './decorators/resolved-view';
 
 function configure(aurelia, configCallback) {
   let config = aurelia.container.get(Config);
 
-  configCallback(config);
+  if (configCallback) {
+    configCallback(config);
```

Comment:
`configCallback(aurelia.container.get(Config));`

It's small enough. And there's no need to fetch the resource if there's no callback supplied.